Add IQS052 rule (#2426)
home-assistant/developers.home-assistant: docs/core/integration-quality-scale/rules/iqs052.md

---

> title: "IQS052 - The documentation provides examples the user can use. For example (snippets of) configuration, automations, blueprints, scripts, or dashboards"

Reviewer: We shouldn't include examples of examples in the rule name.

---

IQS052 is as follows:

The documentation provides examples the user can use. For example (snippets of) configuration, automations, blueprints, scripts, or dashboards

## Reasoning

> If devices are likely going to be used for a few use cases, we can provide blueprints or scripts to help them set it up.

Reviewer: I think we should say that we want a limited set of examples, ie the most common or useful ones that shows how to use the integration. We don't want the docs to replace the forum or blueprint exchange to collect all sorts of examples.

Keeping the docs succinct is important.

---
title: "IQS052 - The documentation provides automation examples the user can use."
---

IQS052 is as follows:

> The documentation provides automation examples the user can use.

Author: I now changed `examples` -> `automation examples` because if I only remove the whole list of examples, it does not really provide any insights of what `examples` mean

## Reasoning

> To show how the integration can be used, we should provide a limited set of blueprints, containing common or useful ones.

Author: But since we want blueprints and scripts (since those are ready to be used) I have a hard time integrating this in this sentence. 

(I also don't reeaally see how dashboards and automations can be shared out of the box in a neat way)

Reviewer: A YAML-example of an automation is also ok, I think.

I don't know much about dashboards. Someone else should give input on that.

This will help users to get started with the integration faster and easier.

The documentation pages should not be used as a collection or as a replacement of the blueprint exchange on the forums.

## Example implementation

```markdown
## Examples

### Turning off the LEDs during the night
The status LEDs on the device can be quite bright.

> To tackle this, we've created a blueprint where you can easily automate the LEDs turning off when the sun goes down.

Reviewer: ```suggestion
To tackle this, you can use this blueprint to easily automate the LEDs turning off when the sun goes down.
```
